manifests/install-no-extension.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Let's call this file `install-with-extension`. By default Argo-Events should be installed without Validating webhook as it is an addon.

Reply:
I made it a separate installation file `install-validating-webhook.yaml`.